library/src/main/kotlin/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -196,7 +209,42 @@ public class ChuckerInterceptor private constructor(
             }
 
         /**
-         * Creates a new [ChuckerInterceptor] instance with values defined in this builder.
+         * Sets a list of [Regex] to skip paths. When any of the [Regex] matches a
+         * request path, the request will be skipped. Include [RegexOption] where
+         * necessary.
+         *
+         * ```
+         * ".*(jpg|jpeg|png|gif|webp|svg|bmp|ico)$".toRegex(), // Ignore all image requests
+         *  ".*iGnOrE.*"toRegex(RegexOption.IGNORE_CASE), // Case insensitive
+         *  ".*path/to/skip.*".toRegex(),
+         *  ".*path/ends/with/dev$".toRegex(),
+         * ```
+         */
+        public fun skipPaths(vararg skipPaths: Regex): Builder =
+            apply {
+                this@Builder.skipPathsRegex.addAll(skipPaths.toSet())
+            }
+
+        /**
+         * Sets a list of [Regex] to skip domains. Domain names are evaluated in
+         * lowercase format. When any of the [Regex] matches a request domain,
+         * the request will be skipped. Include [RegexOption] where necessary.
+         *
+         * ```
+         *  ".*iGnOrE.*"toRegex(RegexOption.IGNORE_CASE),
+         *  "ignoresubdomain.*".toRegex(),
+         *  "domainname.*".toRegex(),
+         *  ".*.dev$".toRegex(),
+         * ```
+         */
+        public fun skipDomain(vararg skipDomain: Regex): Builder =
```

Comment:
If we add this, we'll have to add also `skipDomain(vararg skipDomain: String)`

Also here please remove the vararg

Reply:
vararg to be removed.  
Should we have both string and regex or deprecate string?

Reply:
We can have both 👍 they're so simple no need to deprecate the Regex